src/multiselectMixin.js

```diff
@@ -328,7 +328,7 @@ export default {
         if (this.tagPosition === 'bottom') {
           options.push({ isTag: true, label: normalizedSearch })
```

Comment:
The label here should match what is on line 331

Reply:
Yeah, just updated that now. Thanks